test/Tests/test_dual_names.jl

```diff
@@ -8,21 +8,23 @@
     MOI.set(model, MOI.ConstraintName(), CI{SVF, MOI.GreaterThan{Float64}}(1), "greaterthan1")
     @test model.con_to_name[CI{SAF{Float64}, MOI.LessThan{Float64}}(1)] == "lessthan"
     @test model.con_to_name[CI{SVF, MOI.GreaterThan{Float64}}(1)] == "greaterthan1"
-    # Dualize
+
+    # Dualize without names
     dual_problem = dualize(model)
     dual_model = dual_problem.dual_model
     primal_dual_map = dual_problem.primal_dual_map
     # Query variable names
     vi_1 = primal_dual_map.primal_con_dual_var[CI{SVF, MOI.GreaterThan{Float64}}(1)][1]
     vi_2 = primal_dual_map.primal_con_dual_var[CI{SAF{Float64}, MOI.LessThan{Float64}}(1)][1]
-    @test MOI.get(dual_model, MOI.VariableName(), vi_1) == "greaterthan1_1"
-    @test MOI.get(dual_model, MOI.VariableName(), vi_2) == "lessthan_1"
+    @test MOI.get(dual_model, MOI.VariableName(), vi_1) == ""
+    @test MOI.get(dual_model, MOI.VariableName(), vi_2) == ""
     # Query constraint names
     ci_1 = primal_dual_map.primal_var_dual_con[VI(1)]
     ci_2 = primal_dual_map.primal_var_dual_con[VI(2)]
-    @test MOI.get(dual_model, MOI.ConstraintName(), ci_1) == "x1"
-    @test MOI.get(dual_model, MOI.ConstraintName(), ci_2) == "x2"
+    @test MOI.get(dual_model, MOI.ConstraintName(), ci_1) == ""
+    @test MOI.get(dual_model, MOI.ConstraintName(), ci_2) == ""
```

Comment:
The default for no names is to return a empty string or is it an error?
we should check, maybe test some solver like Gurobi

Reply:
It is an empty string
```julia
using JuMP
model = Model()
@variable(model, x)
c = @constraint(model, x >= 0)
MOI.get(model, MOI.ConstraintName(), c)
```

Reply:
Perfect!